### DISCUSSION TITLE
Where to put shared content between render and main?

### DISCUSSION THREAD

**Discussion by sustainjane98** [saltyshiomix/nextron#189] [Q&A]
I want to create an enum to contain all names of my ipc channel names, but unfortunately I cannot use them inside the renderer process. Anyone has a solution for this?

![Untitled](https://user-images.githubusercontent.com/47473728/121786748-39dfe300-cbc2-11eb-9bec-4518f6e6b027.png)

**Comment by Psycokwet** [comment]
I just made a folder in which i put everything I want to share (default values, ipcs keys, function utilities) in a special folder, and made a script that always copy thie folder in the other process before starting dev or build

No clean solution exist yet in nextron for that, it's an ongoing issue, but, that's does the trick for me :)
